skbio/stats/composition.py

```diff
@@ -1701,3 +1710,203 @@ def _check_orthogonality(basis):
     basis = np.atleast_2d(basis)
     if not np.allclose(basis @ basis.T, np.identity(len(basis)), rtol=1e-4, atol=1e-6):
         raise ValueError("Basis is not orthonormal")
+
+
+def _welch_ttest(x1, x2):
```

Comment:
I thought Welch's _t_-test is already implemented in SciPy, as: `ttest_ind(x, y, equal_var=False)`. But I could be wrong.